src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -22,8 +22,10 @@ const AVATAR_SIZE = 24;
 const CONTAINER_LEFT_PADDING = 60;
 const CONTAINER_RIGHT_PADDING = 10;
 const AVATAR_LEFT_MARGIN = 4;
-const BUBBLE_LEFT_PADDING = 4;
-const BUBBLE_RIGHT_PADDING = 4;
+const BUBBLE_LEFT_PADDING = 8;
+const BUBBLE_RIGHT_PADDING = 8;
+const BUBBLE_LEFT_PADDING_ATTACHMENT = 4;
+const BUBBLE_RIGHT_PADDING_ATTACHMENT = 4;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The constants `BUBBLE_LEFT_PADDING_ATTACHMENT` and `BUBBLE_RIGHT_PADDING_ATTACHMENT` have been introduced to handle padding for attachments. This is a good approach as it allows for different padding values depending on whether there are attachments or not.
```diff
- const BUBBLE_LEFT_PADDING = 4;
- const BUBBLE_RIGHT_PADDING = 4;
+ const BUBBLE_LEFT_PADDING = 8;
+ const BUBBLE_RIGHT_PADDING = 8;
+ const BUBBLE_LEFT_PADDING_ATTACHMENT = 4;
+ const BUBBLE_RIGHT_PADDING_ATTACHMENT = 4;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->